website/orders/templates/orders/shift_header.html

```diff
@@ -5,7 +5,7 @@ <h1>{{ shift.venue.venue.name }}</h1>
         <template v-if="shift.assignees && shift.assignees.length > 0">
             At your service:
             <template v-for="(assignee, index) in shift.assignees">
-                <% assignee %>
+                <% assignee.username %>
```

Comment:
assignee.username is a username like `s4809327`, so that's not what you want

Reply:
Yes but not in the API, `__str__` is mapped to `username` in the API serializer for `User`s. I can rename the field if necessary?

Reply:
maybe that is better code quality